docs/reference/javascript-api-for-office.md

```diff
@@ -13,19 +13,20 @@ The JavaScript API for Office enables you to create web applications that intera
 
   **Outlook** also uses the Common API syntax. Everything under the alias Office contains objects you can use to write scripts that interact with content in Office documents, worksheets, presentations, mail items, and projects from your Office Add-ins. You must use these Common APIs if your add-in will target Office 2013 and later. This object model uses callbacks.
 
-- **Host-specific APIs** - APIs that were introduced with **Office 2016**. This object model provides host-specific strongly-typed objects that correspond to familiar objects that you see when you use Office clients, and represents the future of Office JavaScript APIs. The host-specific APIs currently include the Word JavaScript API and the Excel JavaScript API.
+- **Host-specific APIs** - APIs that were introduced with **Office 2016**. This object model provides host-specific strongly-typed objects that correspond to familiar objects that you see when you use Office clients, and represents the future of Office JavaScript APIs. The host-specific APIs currently include the the Excel JavaScript API, the OneNote JavaScript API, the PowerPoint JavaScript API, and the Word JavaScript API.
```

Comment:
Last sentence: 
- Typo: "the the Excel" -> "the Excel".
- Are these written like "X JavaScript API" for SEO purposes? If not, could we rephrase this to be "The host-specific JavaScript APIs currently include Excel, OneNote, PowerPoint, and Word." ? That feels more concise and easy to parse for the hosts.